src/traccuracy/_tracking_graph.py

```diff
@@ -524,3 +524,23 @@ def set_edge_attribute(self, ids, attr, value=True):
             )
         for _id in ids:
             self.graph.edges[_id][attr] = value
+
+    def get_tracklets(self):
+        """Gets a list of new TrackingGraph objects where each graph is a subgraph of a
```

Comment:
This phrasing is confusing (specifically "each graph is a subgraph of a tracklet"). Perhaps: "Gets a list of new TrackingGraph objects containing all tracklets of the current graph."